src/discussions/posts/post-actions-bar/PostActionsBar.jsx

```diff
@@ -25,30 +25,36 @@ function PostActionsBar({
   const handleCloseInContext = () => {
     postMessageToParent('learning.events.sidebar.close');
   };
+
+  const loadingStatus = useSelector(selectconfigLoadingStatus);
   const blackoutDateRange = useSelector(selectBlackoutDate);
+
   return (
     <div className="d-flex justify-content-end py-1 flex-grow-1">
       {!inContext && (
         <>
           <Search />
-          <div className="border-right border-light-400 mx-3" />
         </>
       )}
       {inContext && (
-        <h4 className="d-flex flex-grow-1 font-weight-bold my-0 py-0 align-self-center">
-          {intl.formatMessage(messages.title)}
-        </h4>
+      <h4 className="d-flex flex-grow-1 font-weight-bold my-0 py-0 align-self-center">
+        {intl.formatMessage(messages.title)}
+      </h4>
       )}
       {
-        !inBlackoutDateRange(blackoutDateRange) && (
-          <Button
-            variant={inContext ? 'plain' : 'brand'}
-            className="my-0"
-            onClick={() => dispatch(showPostEditor())}
-            size="sm"
-          >
-            {intl.formatMessage(messages.addAPost)}
-          </Button>
+        (!inBlackoutDateRange(blackoutDateRange) && loadingStatus === 'successful') && (
```

Comment:
This constant can be used instead of text 'successful' https://github.com/openedx/frontend-app-discussions/blob/72df5ecb23768ae5828be979541a974e09f8585a/src/data/constants.js#L67  

Other then that looks good